pkg/virt-handler/notify-server/server.go

```diff
@@ -154,7 +155,23 @@ func RunServer(virtShareDir string, stopChan chan struct{}, c chan watch.Event,
 	case <-done:
 		log.Log.Info("notify server done")
 	case <-stopChan:
-		grpcServer.Stop()
+		gracefulShutdownTimeout := 10
```

Comment:
Please move all this case handling to a function with a proper name. We wand a reader to understand what is done here without diving into details, it makes it easy to read and understand.